src/Files.App/Commands/Manager/CommandManager.cs

```diff
@@ -563,5 +577,112 @@ private void Action_PropertyChanged(object? sender, PropertyChangedEventArgs e)
 				}
 			}
 		}
+
+		[DebuggerDisplay("Command {Code} (Modifiable)")]
+		private class ModifiableCommand : ObservableObject, IRichCommand
```

Comment:
To move to a new file. The concept is too important to be shove under another class.